test/test_files.py

```diff
@@ -102,6 +102,25 @@ def test_move_already_at_destination(self):
         self.i.move()
         self.assertEqual(self.i.path, old_path)
 
+    def test_move_file_with_colon(self):
```

Comment:
Do these test cases reproduce the original problem (without the fix coded in)?

Reply:
test_move_file_with_colon reproduces the original problem, yes:

```
self = <test.test_files.MoveTest testMethod=test_move_file_with_colon>

    def test_move_file_with_colon(self):
        self.i.artist = u'C:DOS'
        self.i.move()
>       self.assertIn('C_DOS', self.i.path.decode())
E       AssertionError: 'C_DOS' not found in 'C:\\Users\\GUNTHE~1\\AppData\\Local\\Temp\\tmpcmwnupd6\\testlibdir\\C_\\DOS\\two\\three.mp3'
```

The other two don't because their setup eschews the original problem.

Reply:
Awesome, thanks for that. Since I don't program on Windows, I've been trying to find a good way to reproduce stuff like this.